game-core/src/main/java/games/strategy/triplea/UrlConstants.java

```diff
@@ -25,7 +25,7 @@
 
   TRIPLEA_FORUM("https://forums.triplea-game.org/"),
 
-  AXIS_AND_ALLIES_FORUM("https://www.axisandallies.org/forums/index.php"),
+  AXIS_AND_ALLIES_FORUM("https://axisandallies.org/forums"),
```

Comment:
Adding a / at the end of the url might already do the trick.

Reply:
Not really sure about that anymore, it should work nevertheless.
I'll think about this in the next few hours.

Reply:
I originally tried with a / and was getting a 302.